src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/ContaoBackendViewTemplate.php

```diff
@@ -48,4 +91,28 @@ public function get($name)
     {
         return $this->$name;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function translate($string, $domain = null, array $parameters = array(), $locale = null)
+    {
+        if ($this->translator) {
+            return $this->translator->translate($string, $domain, $parameters, $locale);
+        }
+
+        return $string;
```

Comment:
Atm I just return the raw string if no translator is defined. Not sure if we should throw an exception here.